MedscribeUI/src/components/Utilities/ThreeBtnSelector.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from '@mantine/core';
+import { useState, useEffect } from 'react';
+
+interface ThreeBtnSelectorProps {
+  options: string[]; // Array of button labels
```

Comment:
nit: please remove comments. You want to avoid putting comments in production code. It actually is more descriptive to change the variable names. For example the options field can be called buttonLabelOptions!

Reply:
oooo intresting, will do! what does nit mean?